examples/md-flexible/scripts/plotPerTraversalAndParameter.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/python3
+
+import os
+import sys
+import plotly.graph_objects as go
+import re
+import numpy
+
+# THIS SCRIPT NEEDS AT LEAST PYTHON 3.8.
+# However, lesser version will probably fail due to invalid syntax instead of this assertion
+assert sys.version_info >= (3, 8)
+
+# ---------------------------------------------- Input ----------------------------------------------
+
+parameterIdentifierString = 'parameter='
+parameterArg = []
+containsParameterArg = False
+
+# help message
+for arg in sys.argv[1:]:
+    if "--help" in arg:
+        print("Usage: ./plotTuning.py parameter=[number, size, density] [path/To//Output/*.out ...]. Meaning number = number of particles, size = boxSize and density = particle density")
+        print("Please use at least python 3.8.")
+        exit(0)
+    elif parameterIdentifierString in arg:
+        parameterArg = arg.split('=', 1)[1]
+        containsParameterArg = True
+    elif not containsParameterArg and not parameterIdentifierString in arg:
+        print("Please specify the parameter for the x-axis: parameter=[number, size, density]")
+        exit(0)
+
+# take all input files as source for a plot
+if len(sys.argv) > 1:
```

Comment:
better check if the argument is a directory or file, otherwise, this will only work properly if one always goes for the `.out` filename, which is completely arbitrary.